src/common/url.js

```diff
@@ -24,6 +29,7 @@ export const getSandboxOptions = (url: string) => {
   }
 
   result.hideNavigation = url.includes('hidenavigation=1');
+  result.isInProjectView = !url.includes('isFileEval=1');
```

Comment:
It should just go back to isInProjectView to stay consistent and introduce less names. I only changed it since the feature split into 2 concepts. We can use =0 so it only shows in module view.